fastdds_python_examples/HelloWorldExample/HelloWorld.i

```diff
@@ -19,7 +19,15 @@
  * This file was generated by the tool fastddsgen.
  */
 
-%module HelloWorld
+%module(moduleimport="if __import__('os').name == 'nt': import win32api; win32api.LoadLibrary('HelloWorld.dll')\nif __package__ or '.' in __name__:\n    from . import _HelloWorldWrapper\nelse:\n    import _HelloWorldWrapper") HelloWorld
+
+// If using windows in debug, it would try to use python_d, which would not be found.
+%begin %{
+#ifdef _MSC_VER
+#define SWIG_PYTHON_INTERPRETER_NO_DEBUG
+#endif
+#include <exception>
+%}
```

Comment:
We should have a corresponding Fast DDS Gen PR so that the generated `.i` has this as well

Reply:
Done